README.md

```diff
@@ -31,6 +31,7 @@ Theia is an extensible platform to develop full-fledged multi-language Cloud & D
 
 ## Getting Started
 Here you can find guides and examples for common scenarios:
+- [Quick Start Guide](https://github.com/theia-ide/theia/blob/master/doc/Developing.md#prerequisites)
```

Comment:
I would rather replace it with `Develop a new Theia application` or `Build your own IDE` pointing to https://github.com/theia-ide/theia-website/blob/1c4d6fc597dfe65104f5e01ff1a7b3497f12d768/doc/Composing_Applications.md after https://github.com/theia-ide/theia-website/pull/11 gets merged.